src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -668,6 +668,34 @@ bool AletheProofPostprocessCallback::update(Node res,
                            {},
                            *cdp);
     }
+    // ======== Reordering
+    // This rule is translated according to the clauses pattern.
+    // As in the cases of the RESOLUTION, CHAIN_RESOLUTION and EQ_RESOLVE rule
+    // an additional or step might need to be applied to a child.
+    case PfRule::REORDERING:
+    {
+      Node trueNode = nm->mkConst(true);
+      std::vector<Node> new_children = children;
+      if (children[0].getKind() == kind::OR
+          && (args[0] != trueNode || children[0] != args[1]))
+      {
+        std::shared_ptr<ProofNode> childPf = cdp->getProofFor(children[0]);
+        if (childPf->getRule() == PfRule::ASSUME
+            || childPf->getRule() == PfRule::EQ_RESOLVE)
```

Comment:
you'd have to remove this to generalize, right?

Reply:
Yes, I handed this in before the meeting yesterday and wanted to ask before changing this.

Reply:
Done